lib/order.rb

```diff
@@ -0,0 +1,56 @@
+require_relative 'messager'
+
+class Order
+  attr_reader :basket, :menu, :complete, :messager, :processed
+
+  def initialize(menu_instance: menu, messager: Messager)
+    @menu = menu_instance
+    @basket = Hash.new(0)
+    @messager = messager
+    @processed = false
+  end
+
+  def add(item, qty)
+    basket[item] += qty
+    "#{item} x#{qty} added to your basket"
+  end
+
+  def overview
+    return 'Basket is empty' if basket.empty?
+    basket.map { |item, qty| "#{item} x#{qty}: £#{qty * menu.dishes[item].to_i}"}
```

Comment:
Metrics/LineLength: Line is too long. [81/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)